.github/workflows/build.yml

```diff
@@ -32,6 +32,10 @@ jobs:
         run: go build -v ./...
         working-directory: ./examples
 
+      - name: Test examples
```

Comment:
Should we at some point have our workflows here pointing to `meta`? Like we do in Charm (https://github.com/charmbracelet/charm/blob/main/.github/workflows/build.yml#L7)

Reply:
not sure we can: the meta workflow just clone, build and test `./...`, we also do the testing in examples and tutorials... maybe if we change it to allow to pass the working directory 🤔 

anyway, will have a look on it in another PR 👍🏻